models/result.go

```diff
@@ -212,3 +220,23 @@ func GetResult(rid string) (Result, error) {
 	err := db.Where("r_id=?", rid).First(&r).Error
 	return r, err
 }
+
+//INFO send 'Event' to all the active webhooks
```

Comment:
Once we remove the unnecessary calls to `sendWebhooks`, we should be able to remove this function entirely.

Reply:
done